cybox/compat.py

```diff
@@ -0,0 +1,57 @@
+import sys
+
+# Syntax sugar.
+_ver = sys.version_info
+
+#: Python 2.x?
+is_py2 = (_ver[0] == 2)
+
+#: Python 3.x?
+is_py3 = (_ver[0] == 3)
+
+
+if is_py2:
```

Comment:
Yes, the version checking code here predates my use of `six`. I was trying to avoid using it, but ran into problems with the Unicode literals.